db/seeds.rb

```diff
@@ -477,6 +477,18 @@ def create_if_none(klass, count = nil)
   end
 end
 
+seeder.create_if_none(ListingEndorsement) do
+  user = User.last
+  listing = Listing.last
+
+  2.times do
```

Comment:
don't know how smart we want to be here but maybe we can select 5 listings and leave an endorsement by 5 different users?